packages/models/tests/server/organizations_test.coffee

```diff
@@ -14,7 +14,19 @@ describe 'Organization attributes', ->
     organization.save
     expect(organization.description).to.eq('Description')
 
+
   it 'includes createdById', ->
     organization.set('createdById', 'fakeid')
     organization.save
     expect(organization.createdById).to.eq('fakeid')
+
+describe 'Organization truncated description', ->
```

Comment:
I don't know if this needs its own describe. I think it would read better as "Organization" ... "includes name", "truncates description"